app/assets/javascripts/modules/show-gov-uk-verify-hint.js

```diff
@@ -0,0 +1,52 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  var $ = global.jQuery
+
+  GOVUK.Modules.ShowGovUkVerifyHint = function () {
+    this.start = function (element) {
+      if (window.location.href.indexOf("/personal-tax-account/sign-in/prove-identity") > -1) {
+        checkLastSuccessfulIdp(element)
+      }
+    }
+    
+    this.render = function (element, data) {
+      renderHint(element, data)
+    }
+
+    function checkLastSuccessfulIdp (element) {
+      $.ajax({
+        url: 'https://www.signin.service.gov.uk/successful-idp',
+        cache: false,
+        dataType: 'jsonp',
+        timeout: 3000
+      }).then(function(data){
+          renderHint(element, data);
+      }, function(e){console.log("error", e)})
+    }
+
+    function renderHint (element, data) {
+      if (data != null && data['found'] == 'true') {
+        $(element).html(generateHtml(data)).show()
+        $('button:contains("Continue")').addClass('govuk-button--secondary')
+      }
+    }
+
+    function generateHtml(data) {
+      return '\
```

Comment:
I think this could be made more legible by concatenating strings, which is the way I've seen construction of markup done in other pieces of JavaScript on GOV.UK - for example:

```javascript
return '<div class="verify-hint-box">'
  + '<h2 class="govuk-heading-m">'
  + 'Someone recently signed in with '
  + data['displayName']
```